src/Bridges/Constraint/Constraint.jl

```diff
@@ -82,6 +85,9 @@ function add_all_bridges(bridged_model, ::Type{T}) where {T}
     MOIB.add_bridge(bridged_model, VectorFunctionizeBridge{T})
     MOIB.add_bridge(bridged_model, SplitIntervalBridge{T})
     MOIB.add_bridge(bridged_model, QuadtoSOCBridge{T})
+    # We do not add `(R)SOCtoNonConvexQuad` because it starts with a convex
```

Comment:
Is there documentation telling users how to manually turn on this bridge?

Reply:
Good point, I can add docs.

Reply:
Should we documents it in the docstrings of this bridge or in a advanced chapter of MOI.Bridge submodule docs?

Reply:
Maybe [here](http://www.juliaopt.org/MathOptInterface.jl/stable/apireference/#Bridges-1)?

Reply:
ok for me